src/MobManager.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+/**
+ * Keeps track of all the individual mobs in the game
+ */
+class MobManager {
+  constructor() {
+    this.mobs = new Map();
+  }
+
+  /**
+   * @param {string} entityRef
+   * @return {Mob}
+   */
+  getMob(entityRef) {
+    return this.mobs.get(mob.uuid);
```

Comment:
I think this is probably a copy/paste error. This should probably be iterating over the mobs and finding by entityRef. That said what is this getter used for? Should it get all mobs matching that entity ref? `ItemManager` doesn't have a get by entityRef mainly because such a method would bring up some considerations that need to be made about storage.

MobManager/ItemManager could potentially be some very large lists as such doing a linear search through them probably isn't the best, as such some optimizations could be made such as maintaining two internal maps. One which are UUID to Npc instance, another being a map of EntityRef to array of UUIDs currently in the list matching this ref. That turns the getter from an O(n) search to an O(1), much better.

Reply:
Actually I don't even need the getter, I'm most interested on the setter and having the list available to iterate when needed, with specific conditions.
Should I just remove the getter?

Reply:
I think it's easier to remove it now and add it back later if we need it

Reply:
Done!